src/main/java/org/example/coffeeshop/service/UserService.java

```diff
@@ -1,14 +1,16 @@
 package org.example.coffeeshop.service;
 
 import org.example.coffeeshop.entity.User;
+import org.example.coffeeshop.request.RegisterUserRequestDto;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
 import java.util.Optional;
 
 public interface UserService {
 
-    User save(User user);
-
     Optional<User> findByEmail(String email);
 
-
+    String createEmployee(RegisterUserRequestDto registerUserRequestDto, BindingResult result, RedirectAttributes redirectAttributes, Model model);
```

Comment:
1. Not a good solution to return String on employee creation
2. Employee creation should happen in EmployeeService/Controller/Repository not in UserService